src/app/profiles.rs

```diff
@@ -159,7 +159,14 @@ where
                         .routes
                         .into_iter()
                         .filter_map(move |orig| convert_route(orig, retry_budget.as_ref()));
-                    match tx.start_send(routes.collect()) {
+                    let overrides = profile
+                        .dst_overrides
+                        .into_iter()
+                        .filter_map(|orig| convert_dst_override(orig));
+                    match tx.start_send(profiles::Routes {
+                        routes: routes.collect(),
+                        dst_overrides: overrides.collect(),
```

Comment:
Nit/TIOLI: I _might_ split the `Routes` initializer out into a separate `let` binding so this reads a little more linearly?